code/_helpers/crew.dm

```diff
@@ -2,6 +2,15 @@
 /proc/get_historic_crew_total()
 	return GLOB.all_crew.len
 
-//Returns the number of live crewmembers
+//Returns the number of alive humans aboard Ishimura
 /proc/get_living_crew_total()
-	return GLOB.living_crew.len
+	var/crew_count = 0
+	for(var/datum/mind/M in GLOB.living_crew)
+		if(!M || QDELETED(M.current) || !ishuman(M.current))	//regular sanity checks
+			continue
+		var/mob/living/L = M.current
+		if(!L.client || L.stat == DEAD || !isStationLevel(L.z))	//only count alive active humans onboard Ishimura, the rest are marooned
```

Comment:
hmm, on second thought, modifying this proc isnt a good idea, make a wrapper which calls it

Also, afaik, people are automatically removed from the list if killed/deleted, so those checks are unnecessary. likewise with the human check

Make a wrapper version that just checks station level and client
Also consider using ```client.is_afk(time)``` to check if people are sitting idle while still connected